.eslintrc.json

```diff
@@ -14,6 +14,8 @@
   },
   "plugins": ["react", "@typescript-eslint", "react-hooks", "prettier"],
   "rules": {
+    "no-unused-vars": "off",
```

Comment:
This enforces better code. I wouldn't turn it off.

Reply:
This was giving me errors when using functions having a parameter inside an interface, so i disabled this rule and enabled the same rule for typescript in line 18, so it still gives me a feedback when i have unused vars